sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -441,6 +441,20 @@ void gnrc_ipv6_nib_change_rtr_adv_iface(gnrc_netif_t *netif, bool enable)
     }
     gnrc_netif_release(netif);
 }
+
+void gnrc_ipv6_nib_change_rtr_adv_rio_iface(gnrc_netif_t *netif, bool enable)
```

Comment:
Shouldn't this function also depend on `gnrc_ipv6_nib_rio`?

Reply:
Shouldn't make a difference if it isn't used, but I added an `#if IS_USED(MODULE_GNRC_IPV6_NIB_RIO)`